cmd/console.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/kubefirst/kubefirst/configs"
+	"github.com/spf13/cobra"
+	"log"
+	"net/http"
+)
+
+var console = &cobra.Command{
+	Use:   "console",
+	Short: "starts ui server",
+	Long:  "starts app server for the Kubefirst console",
```

Comment:
maybe we could extend the long description a little more, maybe adding that the UI will contain some links, and..